purchase_update_seller_price/README.rst

```diff
@@ -0,0 +1,51 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+
+=================================
+Purchase Update Seller PRice
+=================================
+
+This module add the posibility to update the seller price when a PO is confirmed.
```

Comment:
This module add the posibility to update the seller price every time a PO is confirmed.